sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/CreateTableExec.scala

```diff
@@ -43,7 +43,12 @@ case class CreateTableExec(
   override protected def run(): Seq[InternalRow] = {
     if (!catalog.tableExists(identifier)) {
       try {
-        catalog.createTable(identifier, columns, partitioning.toArray, tableProperties.asJava)
+        val tableInfo = new TableInfo.Builder()
```

Comment:
Are there more places that should be migrated? What about CTAS?

Reply:
Migrated all v2 operators.